api-client/src/endpoints/submitHelpForm.ts

```diff
@@ -1,15 +1,142 @@
 import OfferModel from '../models/OfferModel'
 
 export const MAX_COMMENT_LENGTH = 200
+export type ContactChannel = 'eMail' | 'telephone' | 'personally'
+export type ContactGender = 'male' | 'female' | 'any'
+
+type BuildHelpFormBodyParams = {
+  languageCode: string
+  name: string
+  roomNumber?: string
+  contactChannel: ContactChannel
+  additionalContactInformation?: string
+  contactGender: ContactGender
+  comment: string
+  translate: (text: string) => string
+}
+
+const buildHelpFormBody = ({
+  languageCode,
+  name,
+  roomNumber,
+  contactChannel,
+  additionalContactInformation,
+  contactGender,
+  comment,
+  translate,
+}: BuildHelpFormBodyParams): string => {
+  const anyGenderText = contactGender === 'any' ? translate('contactPersonAnyGender') : undefined
+  const maleGenderText = contactGender === 'male' ? translate('contactPersonGenderMale') : undefined
+  const femaleGenderText = contactGender === 'female' ? translate('contactPersonGenderFemale') : undefined
+  return `
+Sprache: ${languageCode}
+
+Name: ${name}
+Zimmernummer: ${roomNumber ?? '-'}
+So möchte ich kontaktiert werden: ${translate(contactChannel)} ${
+    additionalContactInformation ? `(${additionalContactInformation})` : ''
+  }
+${anyGenderText ?? maleGenderText ?? femaleGenderText}
+
+Beschreibung des Anliegens:
+${comment}
+`
+}
+
+const generateEmail = () => {
+  const RANDOM_NUMBER_LENGTH = 8
+  const array = new Uint8Array(RANDOM_NUMBER_LENGTH)
+  return `support+zammad-ticket-${crypto.getRandomValues(array).join('')}@integreat-app.de`
```

Comment:
Random question, what's the story with the email address? Is that just for filtering?

Reply:
for the `form_submit` endpoint an email adress is required. In case the user doesn't provide an email address we generate a random one. It is not possible to pass anything as email address, as well. 
See here:
https://github.com/zammad/zammad/blob/de0574f09d7f6da7685abf53f6de3ca81c7a3a62/app/controllers/form_controller.rb#L49-L64

Reply:
Aaaah, cool, thanks for the explanation :)